beetsplug/discogs.py

```diff
@@ -200,7 +200,7 @@ def get_albums(self, query):
         query = re.sub(r'(?u)\W+', ' ', query).encode('ascii', "replace")
         # Strip medium information from query, Things like "CD1" and "disk 1"
         # can also negate an otherwise positive result.
-        query = re.sub(br'(?i)\b(CD|disc)\s*\d+', '', query)
+        query = re.sub(b'(?i)\b(CD|disc)\s*\d+', '', query)
```

Comment:
maybe my sleep addled brain isn't ready for this, but maybe the wrong is the plain `''`.  Can you try it without the bytes change and  adding a `b` to `''`